notebooks/official/model_monitoring/batch_prediction_model_monitoring.ipynb

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
<p>ENDPOINT should be set separately, and be referred to as API_ENDPOINT</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>fixed</p>